src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -16,27 +18,26 @@ namespace Microsoft.OData.Core
 #if ODATALIB_ASYNC
     using System.Threading.Tasks;
 #endif
+
+    using Microsoft.OData.Core.JsonLight;

```

Comment:
deleted.

---
In reply to: [108261884](https://github.com/OData/odata.net/pull/779#discussion_r108261884) [](ancestors = 108261884)